src/adapters/worlds-manager.ts

```diff
@@ -67,19 +69,17 @@ export async function createWorldsManagerComponent({
     return await worldsCache.fetch(worldName)
   }
 
-  async function storeAcl(worldName: string, acl: AuthChain): Promise<void> {
-    const content = await worldsCache.fetch(worldName)
+  async function storeWorldMetadata(worldName: string, worldMetadata: Partial<WorldMetadata>): Promise<void> {
+    const content = await storage.retrieve(`name-${worldName.toLowerCase()}`)
+    const metadata: Partial<WorldMetadata> = {}
```

Comment:
tiny suggestion because I was not sure about what was going on here, 

```
const contentMetadata = content ? JSON.parse((await streamToBuffer(await content.asStream())).toString())) : {}
const metadata: Partial<WorldMetadata> = Object.assign({}, contentMetadata, worldMetadata)
```